packages/block-library/src/image/index.php

```diff
@@ -215,6 +216,8 @@ function block_core_image_render_lightbox( $block_content, $block ) {
 	// contain a caption, and we don't want to trigger the lightbox when the
 	// caption is clicked.
 	$p->set_attribute( 'data-wp-on-async--click', 'actions.showLightbox' );
+	$p->set_attribute( 'data-wp-class--hide', 'state.contentImageHide' );
+	$p->set_attribute( 'data-wp-class--show', 'state.contentImageShow' );
```

Comment:
because these represent boolean values, lets call them something like `isContentHidden`, `isContentVisible` ?

Reply:
@madhusudhand Ok that's been changed 👍